ansible/roles/schulcloud-server-provisioning/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+  - name: Schulconnex group provisioning deployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: schulconnex-group-provisioning-deployment.yml.j2
+      state: "{{ 'present' if WITH_SCHULCONNEX_PROVISIONING else 'absent'}}"
+    tags:
+      - deployment
+
+  - name: Schulconnex license provisioning deployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: schulconnex-license-provisioning-deployment.yml.j2
+      state: "{{ 'present' if WITH_SCHULCONNEX_PROVISIONING else 'absent'}}"
+    tags:
+      - deployment
+
+  - name: Schulconnex provisioning configmap
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: schulconnex-provisioning-configmap.yml.j2
+      apply: yes
+    tags:
+      - configmap
+
+  - name: Schulconnex provisioning Secret by 1Password
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: schulconnex-provisioning-onepassword.yml.j2
+    when: ONEPASSWORD_OPERATOR is defined and ONEPASSWORD_OPERATOR|bool
+    tags:
+      - 1password
```

Comment:
Missing condition for `WITH_SCHULCONNEX_PROVISIONING` you have to options set the state or extend the when